StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -91,11 +103,31 @@ extension SavedPaymentMethodCollectionView {
             button.accessibilityLabel = String.Localized.edit
             return button
         }()
+        lazy var defaultBadge: UILabel = {
+            let label = UILabel()
+            label.font = appearance.scaledFont(for: appearance.font.base.regular, style: .caption1, maximumPointSize: 20)
+            label.textColor = appearance.colors.textSecondary
+            label.adjustsFontForContentSizeCategory = true
+            label.text = String.Localized.default_text
+            label.isHidden = true
+            return label
+        }()
 
         fileprivate var viewModel: SavedPaymentOptionsViewController.Selection?
+        var isDefaultPM: Bool = false
 
         var isRemovingPaymentMethods: Bool = false {
             didSet {
+                if showDefaultPMBadge {
+                    if isRemovingPaymentMethods {
+                        activateDefaultBadge()
+                        defaultBadge.isHidden = !isDefaultPM
```

Comment:
We should avoid setting .isHidden directly as it can lead to unexpected behavior if setting it to true multiple times in a row. Instead, it is safer to call our helper function `setHiddenIfNecessary`  on it instead.